src/__stories__/naked-card-story.tsx

```diff
@@ -0,0 +1,202 @@
+            Action
+        </ButtonPrimary>
+    ) : undefined;
+
+    const buttonLink = actions.includes('link') ? (
+        <ButtonLink href="https://google.com">Link</ButtonLink>
+    ) : undefined;
+
+    const onPress = actions.includes('press') ? () => null : undefined;
+
+    const interactiveActions = onPress
+        ? {onPress}
+        : {
+              button,
+              buttonLink,
+          };
+
+    return (
+        <ResponsiveLayout>
+            <Box paddingY={24}>
+                <NakedCard
+                    dataAttributes={{testid: 'naked-card'}}
+                    headline={headline && <Tag type={headlineType}>{headline}</Tag>}
+                    pretitle={pretitle}
+                    title={title}
+                    subtitle={subtitle}
+                    description={description}
+                    media={
+                        media === 'video' ? (
+                            <Video src={VIDEO_SRC} aspectRatio="16:9" dataAttributes={{qsysid: 'video'}} />
+                        ) : media === 'image' ? (
+                            <Image aspectRatio="16:9" src={IMAGE_SRC} />
+                        ) : (
+                            <Image circular src={IMAGE_SRC} />
+                        )
+                    }
+                    {...interactiveActions}
+                    extra={withExtra ? <Placeholder /> : undefined}
+                    onClose={closable ? () => {} : undefined}
+                    actions={
+                        withTopAction
+                            ? [
+                                  {
+                                      Icon: IconMobileDeviceRegular,
+                                      onPress: () => {
+                                          alert('icon press');
+                                      },
+                                      label: 'Device',
+                                  },
+                              ]
+                            : undefined
+                    }
+                />
+            </Box>
+        </ResponsiveLayout>
+    );
+};
+
+Default.storyName = 'NakedCard';
+Default.args = {
+    media: 'image',
+    headlineType: 'promo',
+    headline: 'Priority',
+    pretitle: 'Pretitle',
+    title: 'Title',
+    subtitle: 'Subtitle',
+    description: 'This is a description for the card',
+    withExtra: false,
+    actions: 'button',
+    closable: false,
+    withTopAction: false,
+};
+Default.argTypes = {
+    media: {
+        options: ['image', 'circular image', 'video'],
+        control: {type: 'select'},
+    },
+    headlineType: {
+        options: ['promo', 'active', 'inactive', 'success', 'warning', 'error'],
+        control: {type: 'select'},
+    },
+    actions: {
+        options: ['button', 'link', 'button and link', 'on press', 'none'],
+        control: {type: 'select'},
+    },
+};
+
+type SmallArgs = {
+    media: 'image' | 'circular image' | 'video';
```

Comment:
Video in smallNaked is not allowed on the design side because this card will be used for reduced spaces and the video controls will not fit in many cases. should we remove the video arg from smallNaked story?

wdyt @aweell ?

Reply:
Take into account that there isn't video controls here, as in MediaCard.
From the source side we can not ban video as media, but I can remove it from the story

Reply:
I would remove it from story to avoid the usage

Reply:
only for small? regular NakedCard does allow video?

Reply:
Yes, regular naked allow video

Reply:
done